README.md

```diff
@@ -7,6 +7,7 @@ Simple focus animations for tiling window managers.
 <br>
 
 Compatible with sway as well as all X based window managers (i3, bspwm, awesome-wm, xmonad...).
+For Hyprland, consider [pyt0xic/hyprfocus](https://github.com/pyt0xic/hyprfocus)
```

Comment:
I think its a bit out of place having this in the very first paragraph. This spot in the wiki would probably be the most appropriate place - [prA2uI00%DW8.](https://github.com/fennerm/flashfocus/wiki#alternatives-to-flashfocus). It is a bit out of the way though. If you want to move that to the bottom of the readme and add the hyprfocus recommendation I would approve it.

Reply:
Completely missed the wiki! Unsure how to contribute to wikis, but a simple one-liner at the end will suffice:
https://github.com/laur89/flashfocus/wiki

Reply:
Yeah unfortunately there's no way to make PRs against wikis. I can make the change

Reply:
Done :+1: